packages/data-point/lib/entity-types/entity-collection/factory.test.js

```diff
@@ -24,20 +24,6 @@ describe('parse loose modifiers', () => {
     expect(result.compose).toHaveProperty('type', 'ReducerMap')
     expect(result.compose.reducer).toHaveProperty('type', 'ReducerPath')
   })
-
-  test('modelFactory#create default | checks multiple reducers in an entitiy to have matching properties', () => {
-    const result = modelFactory.create({
-      map: '$a',
-      find: '$a',
-      filter: '$a'
-    })
```

Comment:
this is no longer a valid object, because `map` / `filter` / `find` would need to be inside a `compose`